### ISSUE TITLE
Python3 compatibility

### ISSUE THREAD

**Issue by pyro12** [peno64/service.subtitles.localsubtitle#12]
Any chance this will be updated so it will work with Matrix and beyond?

**Comment by peno64** [comment]
It does work on Matrix...

**Comment by pyro12** [comment]
Apologies.  When I tried to install it, a notification said (copied from log) "The dependency on xbmc.python version 2.7.0 could not be satisfied"  and I immediately assumed it didn't work with Python3.

Upon restarting, there was an update for Pysubs2 available.  I updated that and successfully installed your addon.

Thank you for the addon and your help!

**Comment by peno64** [comment]
Did you install the addon via my repo? https://peno64.github.io/repository.peno64/

**Comment by pyro12** [comment]
I dl'd the repo as a zip and "Install from zip" to install the repo, then "Install from repo",  Peno64/Subtitles/LocalSubtitle.  It's possible I chose 'Opensubs w/ dual 'first as I was interested in both.  I don't recall which I tried first.

So, yes. I installed from your repo.  I strongly dislike the 'File Manger' in Kodi so I simply install repos from zip.  Been doing it this way for years.

**Comment by peno64** [comment]
This is indeed strange. I will have to try this also. It is as it first a wrong version was installed and after restart the correct version. Is that on  kodi 19 or a kodi 20 alfa?

**Comment by pyro12** [comment]
Haha!  I think I have found the issue!  I posted [here](https://github.com/moham96/service.subtitles.opensubtitles_by_opensubtitles/issues/1) about an older version called Dual Subtitles not being able to be installed in Matrix.  I probably had an older version of pysubs2 installed from trying to use that or something...

You posted a fix and It seems you did indeed take over the project! 

It seems I "jumped the gun" posting my issue.  If I'd simply tried "turning it off and back on again" it would have worked.

I'm using the Linux flatpak of Matrix 19.4 because some addons were not playing well with Debian Bullseye's version of Python. 

Thank you for being so attentive to your addon!
